DOMAIN.md

```diff
@@ -0,0 +1,73 @@
+# Domain Models
+
+## Waste Stream
+
+> **Definition**:
+> Represents a specific category of waste material, crucial for matching the disposal needs of customers with the capabilities of service providers.
+> **Role**:
+> Used to classify the waste for collection and processing, ensuring that waste is handled according to environmental standards.
+> **Attributes**:
+> id: Unique identifier for the Waste Stream.
+> label: Human-readable name (e.g., paper, metal, glass), important for customer selection and service provider matching.
+> **Example**: In the 'Service Partners Availability' use case, the customer selects a Waste Stream by id, and the system matches this with available Service Providers who can handle this type of waste on the requested date.
```

Comment:
```suggestion
> **Definition**:
> Represents a specific category of waste material, crucial for matching the disposal needs of customers with the capabilities of service providers.
>
> **Role**:
> Used to classify the waste for collection and processing, ensuring that waste is handled according to environmental standards.
>
> **Attributes**:
> - id: Unique identifier for the Waste Stream.
> - label: Human-readable name (e.g., paper, metal, glass), important for customer selection and service provider matching.
>
> **Example**: In the 'Service Partners Availability' use case, the customer selects a Waste Stream by id, and the system matches this with available Service Providers who can handle this type of waste on the requested date.
```

Otherwise it renders all in one line

<img width="1090" alt="Screenshot 2024-02-07 at 10 28 06" src="https://github.com/seenons/seenons-be-assignment/assets/57448/cba99986-ad5d-4077-83c4-28ecde6d6f18">


Same for all the blocks here

Reply:
Fixed, thanks. I used a plugin locally and didn't notice that GH reads it differently